rrmngmnt/host.py

```diff
@@ -490,3 +491,62 @@ def is_connective(self, tcp_timeout=20.0):
             "Use Host.executor().is_connective() instead."
         )
         return self.executor().is_connective(tcp_timeout=tcp_timeout)
+
+    def set_iptables(
```

Comment:
Firewall module is better, keep in mind that we have iptable and firewalld. if we want to support only iptables now we need to check if the host has iptables and if it is used by the system.